src/components/dropdown.component.js

```diff
@@ -4,15 +4,51 @@ export default class DropdownComponent {
     constructor(query) {
```

Comment:
How do I get the selected value? If I want to change the selection via code, how do accomplish that?

Reply:
I've add a "setSelection" metho to do this.